lib/src/components/servo/service.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/servo/servo.dart';
+import 'package:viam_sdk/src/gen/common/v1/common.pb.dart';
+import 'package:viam_sdk/src/gen/component/servo/v1/servo.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+
+class ServoService extends ServoServiceBase {
+  final ResourceManager _manager;
+
+  ServoService(this._manager);
+
+  Servo _servoFromManager(String name) {
```

Comment:
You may eventually want some level of genericism to allow grabbing any generic resource from the manager; that way you don't have to write the same `try/catch` many times with a resource not found error.

Reply:
Thats a great call out, i'll look into that while building out the next one. Thank you!